roles/networking_mapper/tasks/load_env_definition.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Load the Network Environment Definition from file if not given
+  when: "cifmw_networking_env_definition is not defined"
+  block:
+    - name: Check for Networking Environment Definition file existence
+      ansible.builtin.stat:
+        path: "{{ cifmw_networking_mapper_networking_env_def_path }}"
+      register: _net_env_def_stat
+
+    - name: Load the Networking Definition from file
+      when: "_net_env_def_stat.stat.exists"
```

Comment:
shouldn't we `assert` the file exists instead? Else we may end with a failure later, and it may be hard to find the root cause.

Reply:
@cjeanner In this case, ci_nmstate can be used without the mapper. Anyone can directly pass NMstates though `cifmw_ci_nmstate_instance_config` .

Reply:
so it means:
- add the `assert` in this file
- from the ci_nmstate, check if this `instance_config` is set, if not, call the load_env_defintiion.yml from networking_mapper role

Seems to be the best way to avoid weird, hard to debug issues?